### PR TITLE
Request to join build working group

### PR DESCRIPTION
**Pull request by maclover7**
I've been commenting on a bunch of issues, and have been working on some
Jenkins pipeliness stuff. I'd like to get more involved with the working
group and help share the load!

cc @nodejs/build
cc @refack

### CODE REVIEW

**Comment by refack** [comment]
🥇
+1 from me

cc @nodejs/build (silly GitHub)

File: README.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ missing please open an issue. If you are interested in joining check out
 - Gibson Fahnestock [@gibfahn](https://github.com/gibfahn)
 - Johan Bergström [@jbergstroem](https://github.com/jbergstroem)
 - João Reis [@joaocgreis](https://github.com/joaocgreis)
+- Jon Moss [@maclover7](https://github.com/maclover7)
```
Comment (gibfahn) [inline README.md]: Could you move yourself to between `kunalspathak` and `mhdawson`? I know it's not particularly obvious (https://github.com/nodejs/build/pull/930), but this is sorted by GitHub ID.

**Comment by refack** [comment]
It's markdown so se can add a comment `<!-- sorted by GitHub handle -->`

**Comment by maclover7** [comment]
updated @gibfahn, not sure why I thought it was sorted by first name 😆

**Comment by gibfahn** [comment]
>updated @gibfahn, not sure why I thought it was sorted by first name 😆

Probably because we put the names first 😬

**Comment by gibfahn** [comment]
@nodejs/build any objections? If not we should consider this settled.

**Comment by gibfahn** [comment]
Landed in https://github.com/nodejs/build/commit/c1f91cc4e9620bd6ef39bb7074c402766e50c7da

**Pull request closed without merging**